cds/modules/deposit/static/js/cds_deposit/avc/components/cdsDeposits.js

```diff
@@ -50,7 +56,26 @@ function cdsDepositsCtrl(
   // Show message when window is closing
   this.onExit = false;
 
-  this.overallState = {};
+  // Fetch the latest record data (only if it's a project)
+  this.fetchRecord = function() {
+    // Fetch only if it is in ``draft`` mode
+    if (that.master.metadata._deposit.status === 'draft') {
+      cdsAPI.action(that.master.links.self || that.masterLinks.self , 'GET', {}, jwt).then(function(data) {
+        // Metadata for the project
+        $scope.$broadcast(
+          'cds.deposit.metadata.update.' + data.data.metadata._deposit.id,
+          data.data.metadata
+        );
+        // Metadata for the videos
+        _.each(data.data.metadata.videos, function(_metadata) {
```

Comment:
Maybe `lodash` not needed here, you could just use:
`data.data.metadata.video.forEach()`

Reply:
Just for efficiency, but is true the number of the videos will be very small I'll change it thanks!